docs/how-to/required-software.md

```diff
@@ -0,0 +1,96 @@
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/xcode-extraction.png)
+
+</div>
+
+Once complete, you can launch Xcode from that location, however you can move it to the Applications if you desire by dragging Xcode to Applications (on the left hand side of Finder), however this may require Administrator permission:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/finder-admin-permissions.png)
+
+</div>
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/xcode-in-applications.png)
+
+</div>
+
+
+When launching Xcode for the first time, you might be prompted to enter the administrator's name and password to continue:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode is being downloaded](assets/required-software/xcode-admin.png)
+
+</div>
+
+You'll need to accept the terms and conditions that Apple present in order for Xcode to run:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode's Terms and Conditions](assets/required-software/xcode-terms.png)
+
+</div>
+
+A dialog will be presented showing which additional simulators and SDKs you want to install. Check the box for iOS, watchOS and tvOS can remain unchecked: 
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode SDK installation selection](assets/required-software/xcode-install-sdk.png)
+
+</div>
+
+Xcode will download and install the additional iOS SDK:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode's installation progress](assets/required-software/xcode-installing.png)
+
+</div>
+
+Your Xcode installation and setup is now complete. Depending on your internet speeds, the iOS SDK might be downloading still. Either way, this screen should be showing, along with the `What's New in Xcode` screen:
```

Comment:
Is it worth saying what "this screen" is, e.g. "Either way, the Xcode splash screen should be showing,"?

Reply:
I've gone with `welcome screen` as I personally wouldn't class it as a splash screen. Is that better for you?

Reply:
Yep, welcome screen is good, just a small description of what the screen is was all I was suggesting!